plugins/woocommerce/includes/wc-rest-functions.php

```
@@ -168,6 +168,194 @@ function wc_rest_set_uploaded_image_as_attachment( $upload, $id = 0 ) {
+		}
+		if ( ! empty( $image['name'] ) ) {
+			wp_update_post(
+				array(
+					'ID'         => $attachment_id,
+					'post_title' => $image['name'],
+				)
+			);
+		}
+	}
+
+	if ( $featured_set ) {
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$featured_set'. (undefined)

(UnusedLocalVariable)